frontend/packages/console-app/src/components/quick-starts/__tests__/quick-start-utils.spec.tsx

```diff
@@ -1,9 +1,17 @@
-import { getQuickStartByName } from '../utils/quick-start-utils';
+import { getQuickStartByName, isDisabledQuickStart } from '../utils/quick-start-utils';
 import { allQuickStarts } from '../data/quick-start-test-data';
 
 describe('quick-start-utils', () => {
   it('should return the quick start corresponding to the id', () => {
     const mockID = allQuickStarts[0].metadata.name;
     expect(getQuickStartByName(mockID).metadata.name === mockID).toBe(true);
   });
+
+  it('should filter out disabled quick starts', () => {
+    const mockID = allQuickStarts[0].metadata.name;
+    const quickStart = getQuickStartByName(mockID);
```

Comment:
Why are you using the `getQuickStartByName` here. you know you are picking `allQuickStarts[0]`. Use allQuickStarts[0] whereever needed.

Same for the above unit test.